project/project-info.conf

```diff
@@ -1,6 +1,6 @@
 project-info {
   version: "current"
-  scala-versions: ["2.11", "2.12"]
+  scala-versions: ["2.12", "2.13"]
```

Comment:
I believe we can leave this out altogether, it should be picked up from cross scala versions.

Reply:
Leave out the whole `scala-versions` property?

Reply:
Seems to be when it's dropped :+1: